packages/tasks/src/model-libraries.ts

```diff
@@ -1253,6 +1253,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.zonos,
 		filter: false,
 	},
+	openpeerllm: {
+		prettyLabel: "OpenPeerLLM",
+		repoName: "OpenPeerLLM",
+		repoUrl: "https://huggingface.co/openpeerai/openpeerllm",
+		docsUrl: "https://huggingface.co/OpenPeerAI/OpenPeerLLM/blob/main/README.md",
+		countDownloads: `path_extension:"pt" OR path_extension:"py"`,
```

Comment:
```suggestion
		countDownloads: `path:".meta-huggingface.json"`,
```

would be good to count downloads on a single file, for instance `.meta-huggingface.json`. By doing this, we will count only once when user downloads the repo. If we count every pt and py files, a single clone would count as many downloads.